easybuild/easyconfigs/l/libgeotiff/libgeotiff-1.5.1-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'ConfigureMake'
+
+name = 'libgeotiff'
+version = '1.5.1'
+
+homepage = 'https://directory.fsf.org/wiki/Libgeotiff'
+description = """Library for reading and writing coordinate system information from/to GeoTIFF files"""
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+
+source_urls = ['https://download.osgeo.org/geotiff/libgeotiff']
+sources = [SOURCE_TAR_GZ]
+checksums = ['f9e99733c170d11052f562bcd2c7cb4de53ed405f7acdde4f16195cd3ead612c']
+
+builddependencies = [
+    ('binutils', '2.34'),
+]
+
+dependencies = [
+    ('PROJ', '7.0.0'),
+    ('libjpeg-turbo', '2.0.4'),
```

Comment:
Please list LibTIFF and zlib here as they are direct dependencies and explicitly referenced below

Reply:
@akesandgren ok, done